src/App/App.tsx

```diff
@@ -273,9 +276,19 @@ export const App: FunctionComponent = () => {
                 closable={tab.type !== 'admin'}
               >
                 {tab.type === 'admin' ? (
-                  <AdminTab onOpenMetabaseTab={openMetabaseTab} onOpenImportDataTab={openImportDataTab} />
+                  <AdminTab
+                    showMetabaseHint={showMetabaseHint}
+                    onOpenMetabaseTab={openMetabaseTab}
+                    onOpenImportDataTab={openImportDataTab}
+                  />
                 ) : tab.type === 'import' ? (
-                  <ImportDataTab isActive={tab.id === activeTab} />
+                  <ImportDataTab
+                    isActive={tab.id === activeTab}
+                    closeImportTab={() => {
+                      onEdit(tab.id, 'remove');
+                      setActiveTab(tabs[0].id);
```

Comment:
I just realized that forcefully changing focus to the admin tab when an import completes might be pretty annoying if the user was trying to read the documentation or import another file during that time.

Reply:
good point, I think we should `setActiveTab` only when it's currently at the removed one.

Reply:
Yes, that would make sense.